packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.scss

```diff
@@ -55,6 +48,12 @@
   border-left: 4px solid #6929c4;
 }
 
+.tabletTile {
```

Comment:
Shouldn't `.tabletTile` be removed from the previous class if we're just adding a subset of properties here. Alternatively, if we need the styles above, we should collapse `.desktopTile` and `.tabletTile` into `.tile` applied unconditionally.